scripts/drone/steps/lib.star

```diff
@@ -1255,3 +1255,30 @@ def slack_step(channel, template, secret):
             "template": template,
         },
     }
+
+def verify_grafanacom_step():
+    return {
+        "name": "verify-grafanacom",
+        "image": images["node"],
+        "environment": {
+            "GCOM_API_KEY": from_secret("grafana_api_key"),
+            "GCP_KEY": from_secret("gcp_grafanauploads"),
+        },
+        "commands": [
+            # Download and install `curl` - it isn't available inside of the `node:{version}-alpine` docker image.
+            "apk add curl",
+
+            # Download/install gcom utility
+            "curl -L -o /usr/local/bin/gcom https://github.com/grafana/gcom/releases/latest/download/gcom-linux-amd64",
+            "chmod +x /usr/local/bin/gcom",
+
+            # Authenticate gcloud
+            "printenv GCP_KEY > /tmp/key.json",
+            "gcloud auth activate-service-account --key-file=/tmp/key.json",
+            "./drone/verify-grafanacom.sh",
+
+            # Remove the temporary key file
+            "rm /tmp/key.json",
```

Comment:
```suggestion

            # Remove the temporary key file
            "rm /tmp/key.json",
```

No sense in wasting CPU cycles deleting files that will be deleted later anyways :P

Reply:
Touché. Removed.